fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerMutualTest.java

```diff
@@ -451,6 +455,132 @@ void testMutualIndexingCrashAndAllowContinueNonMutually(boolean explicit) {
+
+        FDBRecordStoreTestBase.RecordMetaDataHook hook = allIndexesHook(indexes);
+        openSimpleMetaData(hook);
+        disableAll(indexes);
+
+        // Build as mutual, crash half way
+        final String testThrowMsg = "Intentionally crash during test";
+        try (OnlineIndexer indexBuilder = newIndexerBuilder(indexes)
+                .setLimit(10)
+                .setConfigLoader(old -> {
+                    throw new RecordCoreException(testThrowMsg);
+                })
+                .build()) {
+            RecordCoreException e = assertThrows(RecordCoreException.class, indexBuilder::buildIndex);
+            assertTrue(e.getMessage().contains(testThrowMsg));
+        }
+
+        // Continue as mutual
+        final FDBStoreTimer timer = new FDBStoreTimer();
+        try (OnlineIndexer indexBuilder = newIndexerBuilder(indexes, timer)
+                .setIndexingPolicy(mutualTakeOverIndexingPolicy(explicit, true)
+                        .setMutualIndexing())
+                .build()) {
+            indexBuilder.buildIndex(true);
+        }
+
+        // validate
+        assertReadable(indexes);
+        assertAllValidated(indexes);
+    }
+
+    @ParameterizedTest
+    @BooleanSource
+    void testSingleTargetContinueAsMutual(boolean explicit) {
+        // Start building with a single target, stop, continue as mutual
+        // Make sure that the mutual indexing is unblocked
+        // Note that a single target session produces a different indexing type stamp than a multi target session
+        final Index index = new Index("indexA", field("num_value_2"), EmptyKeyExpression.EMPTY, IndexTypes.VALUE, IndexOptions.UNIQUE_OPTIONS);
+        List<Index> indexes = List.of(index);
+
+        int numRecords = 32;
+        populateData(numRecords);
+
+        FDBRecordStoreTestBase.RecordMetaDataHook hook = allIndexesHook(indexes);
+        openSimpleMetaData(hook);
+        disableAll(indexes);
+
+        // Build as mutual, crash half way
+        final String testThrowMsg = "Intentionally crash during test";
+        try (OnlineIndexer indexBuilder = newIndexerBuilder(index)
+                .setLimit(5)
+                .setConfigLoader(old -> {
+                    throw new RecordCoreException(testThrowMsg);
+                })
+                .build()) {
+            RecordCoreException e = assertThrows(RecordCoreException.class, indexBuilder::buildIndex);
+            assertTrue(e.getMessage().contains(testThrowMsg));
+        }
+
+        // Continue as mutual
+        try (OnlineIndexer indexBuilder = newIndexerBuilder(index)
+                .setIndexingPolicy(mutualTakeOverIndexingPolicy(explicit, true)
+                        .setMutualIndexing())
+                .build()) {
+            indexBuilder.buildIndex(true);
+        }
+
+        // validate
+        assertReadable(indexes);
+        assertAllValidated(indexes);
+    }
+
+    @ParameterizedTest
+    @BooleanSource
```

Comment:
This doesn't use the parameter
```suggestion
    @Test
```

Reply:
✅